Formula/zookeeper.rb

```diff
@@ -74,7 +74,8 @@ def install
     inreplace "conf/zoo_sample.cfg",
               /^dataDir=.*/, "dataDir=#{var}/run/zookeeper/data"
     cp "conf/zoo_sample.cfg", "conf/zoo.cfg"
-    (etc/"zookeeper").install ["conf/zoo.cfg", "conf/zoo_sample.cfg"]
+    cp "conf/log4j.properties", "conf/log4j.properties.sample"
+    (etc/"zookeeper").install ["conf/zoo.cfg", "conf/zoo_sample.cfg", "conf/log4j.properties.sample"]
```

Comment:
I think this will only ever write on the first install though. Since brew blocks `/etc/` installed files if they already exist AFAIK

Reply:
Is it better to install sample files to libexec/"conf" like this?
```ruby
    cp "conf/zoo_sample.cfg", "conf/zoo.cfg"
    inreplace "conf/zoo.cfg",
              /^dataDir=.*/, "dataDir=#{var}/run/zookeeper/data"
    (etc/"zookeeper").install "conf/zoo.cfg"

    cp "conf/log4j.properties", "conf/log4j.properties.sample"
    (libexec/"conf").install ["conf/zoo_sample.cfg", "conf/log4j.properties.sample"]

```

Reply:
Not sure `libexec` is a good place to install sample configs. Maybe `pkgshare`.

Reply:
Thank you for review.
I made `pkgshare/"examples"` directory and move original sample configs under it.
How about it?